src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerTestingControllerTest.java

```diff
@@ -78,9 +81,9 @@ public void should_save_ccd_event_using_test_rest_endpoints() {
     @Test
     public void should_update_ccd_event_using_test_rest_endpoints() {
         String messageId = randomMessageId();
-        String caseIdForTask = UUID.randomUUID().toString();
+        String caseIdForTask = RandomStringUtils.randomNumeric(16);
```

Comment:
Just for my curiosity could this cause more issues down the line if the case actually exists and belongs to another jurisdiction or same jurisdiction ?